src/ui/components/post/post_listing.rs

```diff
@@ -14,23 +14,14 @@ use leptos::*;
 use leptos_router::*;
 
 #[component]
-pub fn PostListing(post_view: PostView) -> impl IntoView {
+pub fn PostListing<'a>(post_view: &'a PostView) -> impl IntoView {
```

Comment:
This is one area that I don't see the benefit of using a reference since the post view isn't reused in either the post listings or post page.

Reply:
It also prevents cloning wherever this is used.